### DISCUSSION TITLE
Is development ongoing?

### DISCUSSION THREAD

**Discussion by ghost** [jeffvli/sonixd#402] [General]
Hello, was wondering if the development of sonixd is continuing or if it has paused. I haven't seen a lot of action lately.
I love the software and hope to see it flourish.

**Comment by jeffvli** [comment]
It's currently somewhat paused as I'm working on its successor(?) in Feishin which should have its first alpha release soon. https://github.com/jeffvli/sonixd/issues/332

![image](https://user-images.githubusercontent.com/42182408/202547335-b86422de-ae05-44c3-a741-edd000ac165a.png)

**Comment by ghost** [comment]
That's great news. Thanks!

On 17/11/2022 21:02, Jeff wrote:
>
> It's currently somewhat paused as I'm working on its successor(?) in 
> Feishin which should have its first alpha release soon. #332 
> <https://github.com/jeffvli/sonixd/issues/332>
>
> image 
> <https://user-images.githubusercontent.com/42182408/202547335-b86422de-ae05-44c3-a741-edd000ac165a.png>
>
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/jeffvli/sonixd/discussions/402#discussioncomment-4171131>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AGYIPIDCCX5JNZCDEPEEKH3WI2FONANCNFSM6AAAAAASDW6YKA>.
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>
>
-- 
Adrienne Teicher /she/her/
Transfeminine sound and embodied performance artist
www.adrienneteicher.com <https://www.adrienneteicher.com> www.hyenaz.com 
<https://www.hyenaz.com>
